charts/kusion/templates/post-run-job.yaml

```diff
@@ -30,57 +31,20 @@ spec:
           command:
             - /bin/sh
             - -c
-            - >
-              echo "Calling Kusion API for Module Registration..."; 
+            - |
+              echo "Calling Kusion API for Module Registration..."
 
-              echo "Registering kam module...";
-              curl -X POST "${KUSION_SERVER_ENDPOINT}/api/v1/modules" \
+              echo "Registering kam module..."
+              curl -X POST "http://${KUSION_SERVER_ENDPOINT}/api/v1/modules" \
                 -H "Content-Type: application/json" \
-                -d '{"name":"kam","url":"oci://ghcr.io/kusionstack/kam"}';
+                -d '{"name":"kam","url":"oci://ghcr.io/kusionstack/kam","description":"The kam module represents a Kusion opinionated abstraction of the core concepts during application delivery.","doc":"https://www.kusionstack.io/docs/reference/modules/developer-schemas/app-configuration"}'
 
-              curl -X PUT "${KUSION_SERVER_ENDPOINT}/api/v1/modules/kam" \
+              echo "Registering service module..."
+              curl -X POST "http://${KUSION_SERVER_ENDPOINT}/api/v1/modules" \
                 -H "Content-Type: application/json" \
-                -d '{"name":"kam","description":"The kam module represents a Kusion opinionated abstraction of the core concepts during application delivery."}';
+                -d '{"name":"service","url":"oci://ghcr.io/kusionstack/service","description":"The service module represents a kind of workload profile that describes how to run your application code.","doc":"https://www.kusionstack.io/docs/reference/modules/developer-schemas/workload/service"}'
 
-              curl -X PUT "${KUSION_SERVER_ENDPOINT}/api/v1/modules/kam" \
+              echo "Registering network module..."
+              curl -X POST "http://${KUSION_SERVER_ENDPOINT}/api/v1/modules" \
```

Comment:
Adding 'http://' prefix to the endpoint is a good security practice, but ensure the KUSION_SERVER_ENDPOINT variable doesn't already include it to avoid double protocol specification.